test-configs.yaml

```diff
@@ -1040,11 +1040,32 @@ device_types:
     class: arm64-dtb
     boot_method: uboot
 
+# sun50i-h6-pine-h64 model A
   sun50i-h6-pine-h64:
     mach: allwinner
     class: arm64-dtb
     boot_method: uboot
 
+  sun50i-h6-pine-h64-model-b:
+    mach: allwinner
+    class: arm64-dtb
+    boot_method: uboot
+
+#Pine H64 exists in two version A and B.
+#Before 5.5, sun50i-h6-pine-h64 was for model B but could be also used for model A.
+#Starting with 5.5, sun50i-h6-pine-h64 will be only for model A and a new
+#dtb will exists for model B.
+#And so a new LAVA device-type will be necessary.
+#But we need to test model B before 5.5 using the pre 5.5 sun50i-h6-pine-h64.dtb WITH the the new model B LAVA device-type.
+  sun50i-h6-pine-h64-model-b_5.4:
+    base_name: "sun50i-h6-pine-h64-model-b"
+    dtb: "allwinner/sun50i-h6-pine-h64.dtb"
+    mach: allwinner
+    class: arm64-dtb
+    boot_method: uboot
+    filters:
+      - whitelist: {kernel: ['v4.9', 'v4.14', 'v4.19', 'v5.3', 'v5.4']}
```

Comment:
Actually I think there's a problem with this, as a `whitelist` filter needs the strings to match exactly.  So this won't work with stable branches.  Instead, a `regex` filter should be used to allow kernel version strings such as `v5\.4\..*`.